src/main/java/com/algoritm/SecondHigher.java

```diff
@@ -2,6 +2,18 @@
 
 public class SecondHigher {
     public static int find(int[] unsortedArray) {
-        return 0;
+        int firstNumber = unsortedArray[0];
+        int secondNumber = unsortedArray[0];
+
+        for(int i = 1; i < unsortedArray.length; i++) {
```

Comment:
Crees que se pueda implementar un enhanced for loop aquí?

Reply:
Hice el cambio de Array a List